creator-node/src/services/stateMachineManager/stateReconciliation/StateReconciliationQueue.js

```diff
@@ -82,62 +71,60 @@ class StateReconciliationQueue {
    * @param {Function<queue, failedJob>} params.jobFailureCallback the function to call when a job fails
    * @param {Function<job>} params.processManualSync the function to call when processing a manual sync job from the queue
    * @param {Function<job>} params.processRecurringSync the function to call when processing a recurring sync job from the queue
-   * @param {Function<job>} params.processIssueSyncRequests the function to call when processing an issue-sync-requests job from the queue
+   * @param {Function<job>} params.processEnqueueSyncRequests the function to call when processing an issue-sync-requests job from the queue
    * @param {Function<job>} params.processUpdateReplicaSet the function to call when processing an update-replica-set job from the queue
    */
-  registerQueueEventHandlersAndJobProcessor({
+  registerQueueEventHandlersAndJobProcessors({
     queue,
     processManualSync,
     processRecurringSync,
-    processIssueSyncRequests,
+    processEnqueueSyncRequests,
     processUpdateReplicaSets
   }) {
     // Add handlers for logging
     queue.on('global:waiting', (jobId) => {
-      this.log(`Queue Job Waiting - ID ${jobId}`)
+      logger.info(`Queue Job Waiting - ID ${jobId}`)
     })
     queue.on('global:active', (jobId, jobPromise) => {
-      this.log(`Queue Job Active - ID ${jobId}`)
+      logger.info(`Queue Job Active - ID ${jobId}`)
     })
     queue.on('global:lock-extension-failed', (jobId, err) => {
-      this.logError(
+      logger.error(
         `Queue Job Lock Extension Failed - ID ${jobId} - Error ${err}`
       )
     })
     queue.on('global:stalled', (jobId) => {
-      this.logError(`stateMachineQueue Job Stalled - ID ${jobId}`)
+      logger.error(`stateMachineQueue Job Stalled - ID ${jobId}`)
     })
     queue.on('global:error', (error) => {
-      this.logError(`Queue Job Error - ${error}`)
+      logger.error(`Queue Job Error - ${error}`)
     })
 
     // Add handlers for when a job fails to complete (or completes with an error) or successfully completes
     queue.on('completed', (job, result) => {
-      this.log(
-        `Queue Job Completed - ID ${job?.id} - Result ${JSON.stringify(
-          result
-        )}. Queuing another job...`
+      logger.info(
+        `Queue Job Completed - ID ${job?.id} - Result ${JSON.stringify(result)}`
       )
     })
     queue.on('failed', (job, err) => {
-      this.logError(`Queue Job Failed - ID ${job?.id} - Error ${err}`)
+      logger.error(`Queue Job Failed - ID ${job?.id} - Error ${err}`)
     })
 
     // Register the logic that gets executed to process each new job from the queue
     queue.process(
-      JOB_NAMES.HANDLE_MANUAL_SYNC_REQUEST,
+      JOB_NAMES.ISSUE_MANUAL_SYNC_REQUEST,
```

Comment:
per our slack discussion, maybe can change these job names for clarity - can do separately

Reply:
this one actually was the change for clarity -- I renamed to "issue" because it's just issuing the sync request and the other node performs the sync

Reply:
sweet! sg